src/features/user/ProfileV2/components/MicroComponents/MyforestMap/CustomButton.tsx

```diff
@@ -0,0 +1,73 @@
+import PlantedTreesButton from '../../ProjectDetails/PlantedTreesButton';
+import RestoredButton from '../../ProjectDetails/RestoredButton';
+import ConservationButton from '../../ProjectDetails/ConservationButton';
+import DonationInfo from '../../ProjectDetails/DonationInfo';
```

Comment:
As these components are used only by CustomButton, would it make sense to locate them together in a common folder?